modules/openapi-generator/src/main/resources/dart2/serialization/native/native_enum.mustache

```diff
@@ -7,7 +10,7 @@ class {{{classname}}} {
   final {{{dataType}}} value;
 
   @override
-  String toString() => value{{^isString}}.toString(){{/isString}};
+  String toString() => {{#isString}}value{{/isString}}{{^isString}}'$value'{{/isString}};
```

Comment:
I think the old way is better here, `String toString() => value{{^isString}}?.toString(){{/isString}} ?? '';`

Reply:
Sure. There's no need for `?? ''` since the app would crash before this is evaluated.

Reply:
We cannot use `value?` since we must support Dart 2.0. I'll fix the code.